services/121-service/test/payment/do-payment-fsp-onafriq.test.ts

```diff
@@ -0,0 +1,242 @@
+import { HttpStatus } from '@nestjs/common';
+
+import { FinancialServiceProviders } from '@121-service/src/fsps/enums/fsp-name.enum';
+import { TransactionStatusEnum } from '@121-service/src/payments/transactions/enums/transaction-status.enum';
+import { RegistrationStatusEnum } from '@121-service/src/registration/enum/registration-status.enum';
+import { SeedScript } from '@121-service/src/scripts/enum/seed-script.enum';
+import { LanguageEnum } from '@121-service/src/shared/enum/language.enums';
+import { waitFor } from '@121-service/src/utils/waitFor.helper';
+import {
+  doPayment,
+  getTransactions,
+  waitForPaymentTransactionsToComplete,
+} from '@121-service/test/helpers/program.helper';
+import {
+  awaitChangeRegistrationStatus,
+  importRegistrations,
+} from '@121-service/test/helpers/registration.helper';
+import {
+  getAccessToken,
+  resetDB,
+} from '@121-service/test/helpers/utility.helper';
+
+describe('Do payment to 1 PA', () => {
```

Comment:
```suggestion
describe('Do payment to 1 PA with Fsp Onafriq', () => {
```

Unfortunately if you update this you also need to update the snapshots